packages/gatsby-plugin-graphql-codegen/.gitignore

```diff
@@ -2,4 +2,5 @@
 gatsby-node.js
 gatsby-node.d.ts
 graphql-codegen.config.js
-graphql-codegen.config.d.ts
\ No newline at end of file
+graphql-codegen.config.d.ts
+utils
```

Comment:
why is this gitignored?

Reply:
Now this is why having a second pair of eyes is so valuable, this is probably meant for `.npmignore` but my 2am brain confused it with `.gitignore`, which is wrong anyway 🤦‍♀️